### PR TITLE
Low hanging fruit

### PR DESCRIPTION
**Pull request by tymees**
This PR addresses #408 #397 and #399 (and a small warning during messages creation).

Regarding #408:

After a discussion with Michael I decided to make ``HomeView`` a login protected page and move all non-authenticated stuff from that view into the login template. This simplifies the ``HomeView`` template, and reduces a click for users. 

Support emails are now also added to the login template, with a basic anti-spambot protection.

Regarding #397: Basically, added the right validator, removed the character length constraint and added the wordcount script to self-assessment. (With a small tweak such that the wordcount now displays _below_ the textarea for a better visual layout).

Regarding #399: pretty self-explanatory 

Regarding the warning during messages creation: there was an empty translated string in the proposal diff template. That's bad. Changing it out for a plain empty string got rid of the warning with no change in functionality

### CODE REVIEW

File: main/templates/registration/login.html [hunk 1]
```diff
@@ -1,11 +1,70 @@
...
+            <p>
+                <strong>{% trans 'Secretaris FETC-GW' %}</strong><br/>
+                <span class="email" data-email="fetc-gw">
+                    <i>{% trans "Om hier een e-mail adres te zien moet Javascript aan staan" %}</i>
+                </span>
+            </p>
+            <p>
+                <strong>{% trans 'Technische ondersteuning' %}</strong><br/>
+                <span class="email" data-email="portalsupport.gw">
+                    <i>{% trans "Om hier een e-mail adres te zien moet Javascript aan staan" %}</i>
```
Comment (miggol) [inline main/templates/registration/login.html:62]: So thoughtful

**Review by miggol — APPROVED** [review]
Very nice, attentive as always, Ty. Couldn't find anything missing.

It's not within the scope of the issues but it would be nice to have the system messages display on the login page as well.

File: main/templates/registration/login.html (hunk 1 above)
Comment (tymees) [reply main/templates/registration/login.html:62]: I wish I could take the credit; the idea is from USCKI's website lol

**Comment by tymees** [comment]
> It's not within the scope of the issues but it would be nice to have the system messages display on the login page as well.

Yeah, I was thinking the same thing when doing that commit. However, as the View in question is provided by Django itself it was not as simple as copying some stuff. It's possible by using a custom version of said View, but that's not in the 'low-hanging fruit' category lol

I'll create an issue for it

**Pull request merged**